api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -9280,5 +9280,23 @@
 			<column name="sort_weight" type="double"></column>
 		</addColumn>
 	</changeSet>
+
+
+	<changeSet id="201604030003" author="vishnuraom">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<sqlCheck expectedResult="0">SELECT count(*) FROM drug_order WHERE dosing_type = 'SIMPLE' OR dosing_type = 'FREE_TEXT';</sqlCheck>
+			</not>
+		</preConditions>
+		<comment>Converting values in drug_order.dosing_type column(TRUNK-4845)</comment>
+		<update tableName="drug_order">
```

Comment:
I think since it is an update query and the precondition is an either or check, then the whole changeset is good enough instead of  being split into two